integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMultiDomainModelsWithLoadBalancer.java

```diff
@@ -146,7 +146,7 @@ class ItMultiDomainModelsWithLoadBalancer {
   private static final String WLDF_OPENSESSION_APP = "opensessionapp";
   private static final String WLDF_OPENSESSION_APP_CONTEXT_ROOT = "opensession";
   private static final String wlSecretName = "weblogic-credentials";
-  private static final String DATA_HOME_OVERRIDE = "/u01/oracle/mydata";
+  private static final String DATA_HOME_OVERRIDE = "/u01/mydata";
```

Comment:
This is for dataHome override tests. The dataHome is set to /u01/oracle/mydata on purpose. The test will fail if it is set to /u01/mydata.